RCSwitch.cpp

```diff
@@ -86,16 +86,16 @@ enum {
 };
 
 #if not defined( RCSwitchDisableReceiving )
-unsigned long RCSwitch::nReceivedValue = 0;
-unsigned int RCSwitch::nReceivedBitlength = 0;
-unsigned int RCSwitch::nReceivedDelay = 0;
-unsigned int RCSwitch::nReceivedProtocol = 0;
-int RCSwitch::nReceiveTolerance = 60;
 const unsigned int RCSwitch::nSeparationLimit = 4600;
+volatile unsigned long RCSwitch::nReceivedValue = 0;
+volatile unsigned int RCSwitch::nReceivedBitlength = 0;
+volatile unsigned int RCSwitch::nReceivedDelay = 0;
+volatile unsigned int RCSwitch::nReceivedProtocol = 0;
+volatile int RCSwitch::nReceiveTolerance = 60;
 // separationLimit: minimum microseconds between received codes, closer codes are ignored.
```

Comment:
It would be good if the commit message also explained *why* this change was made. You certainly had a reason, but it's not clear what that reason is for a code reviewer.